test/usergroup.routes.test.js

```diff
@@ -42,51 +59,29 @@ describe('usergroup.routes.test.js', () => {
   });
 
   // Create users.
-  beforeEach((done) => {
-    models.User.create(utils.data('user1')).done((e, u) => {
-      expect(e).to.not.exist;
-      user = u;
-      done();
-    });
-  });
-
-  beforeEach((done) => {
-    models.User.create(utils.data('user2')).done((e, u) => {
-      expect(e).to.not.exist;
-      user2 = u;
-      done();
-    });
-  });
-
-  beforeEach((done) => {
-    models.User.create(utils.data('user3')).done((e, u) => {
-      expect(e).to.not.exist;
-      user3 = u;
+  beforeEach(function(done) {
```

Comment:
no, otherwise `this.timeout` doesn't work because `this` is not bind to the right context.